subsys/bluetooth/mesh/Kconfig

```diff
@@ -94,67 +94,95 @@ endif # BT_MESH_RELAY
 endif
 
 config BT_MESH_NLC_PERF_CONF
-	bool "Enable NLC performance configurations"
+	bool "NLC performance configurations"
 	select BT_MESH_COMP_PAGE_2
 	select BT_MESH_COMP_PAGE_1
 	help
-	  Enables performance configurations required for NLC profiles.
-	  When implementing the NLC Basic Lightness Controller profile
-	  individually or in conjunction with other NLC profiles, also
-	  enable CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL.
+	  Enables the framework for NLC profile performance configurations.
+	  Enable individual profiles from the "NLC Profile performance configuration" menu.
+	  Multiple NLC profiles can be combined on a single device.
+	  When combining profiles, the device uses the highest minimum
+	  requirements among all enabled profiles, as defined in the NLC Profile specifications.
 
-choice BT_MESH_NLC_PERF_OPTIONS
-	prompt "NLC configuration options"
-	depends on BT_MESH_NLC_PERF_CONF
+if BT_MESH_NLC_PERF_CONF
+
+config BT_MESH_NLC_PERF_BASELINE
+	bool
 
+config BT_MESH_NLC_PERF_STRICT
+	bool
+
+# Deprecated option. This option is kept for backward compatibility.
 config BT_MESH_NLC_PERF_DEFAULT
-	bool "Default"
+	bool "Default NLC profile performance configuration"
+	default y
```

Comment:
I recommend removing `default y` and add to each sample those are NLC compatible new KConfig options that you added here.